tests/integration/fileTests.py

```diff
@@ -33,6 +34,7 @@ def setUpClass(cls):
         with createApp().app_context():
             # get the submission test user
             sess = GlobalDB.db().session
+            cls.session = sess
```

Comment:
yes, we decided to move the creation of the submission into the test itself for this particular test so we could have it be at least slightly more autonomous and there would be less searching involved to find everything it was associated with. We couldn't pass sess to a random test so we had to make it a class variable.